<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Robot_Setting</class>
 <widget class="QWidget" name="Form_Robot_Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,2">
   <item>
    <widget class="QFrame" name="frame_table">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QTableWidget" name="tableWidget_robotMessage"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_serial">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,1,6,2">
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1,1,1">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>串口号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_serialName"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_serialSearch">
          <property name="text">
           <string>查找串口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_serialOpen">
          <property name="text">
           <string>打开串口</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget_serialData"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1">
        <item>
         <widget class="QPushButton" name="pushButton_getMSG">
          <property name="text">
           <string>获取信息</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_setMSG">
          <property name="text">
           <string>写入信息</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
